Documentation/operations/upgrade.rst

```diff
@@ -321,6 +323,9 @@ Helm Options
 * Deprecated options ``proxy.prometheus.enabled`` and ``proxy.prometheus.port`` have been removed.
   Please use ``envoy.prometheus.enabled`` and ``envoy.prometheus.port`` instead.
 * The unused helm option ``proxy.sidecarImageRegex`` has been removed.
+* Helm value ``endpointStatus`` has been removed. Instead of relying on additional statuses in CiliumEndpoints CRD,
+  please rely on Cilium's metrics to monitor status of endpoints, like ``cilium_policy``, ``cilium_policy_endpoint_enforcement_status``, ``cilium_controllers_failing``, ``cilium_endpoint_state``.
+  More detailed information about specific endpoint status is still available through ``cilium-dbg endpoint get``.
```

Comment:
Grammar and word choice suggestions:

```suggestion
* The helm option ``endpointStatus`` has been removed. Instead of relying on additional statuses in the CiliumEndpoints CRD,
  rely on Cilium's metrics to monitor the status of endpoints. Example metrics include: ``cilium_policy``, ``cilium_policy_endpoint_enforcement_status``, ``cilium_controllers_failing`` and ``cilium_endpoint_state``.
  More detailed endpoint status information is still available through ``cilium-dbg endpoint get``.
```

Reply:
Done.